xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-api/src/main/java/org/xwiki/ratings/internal/SeparatePageRating.java

```diff
@@ -212,7 +212,7 @@ public void save() throws RatingsException
             document.setContentDirty(false);
             document.setCreatorReference(superadmin);
             document.setAuthorReference(superadmin);
-            context.getWiki().saveDocument(getDocument(), context);
+            context.getWiki().saveDocument(getDocument(), context.getMessageTool().get("rating.saveComment"), true, context);
```

Comment:
I agree with @Enygma2002, but this is not a component so I guess you'll have to pass the localization manager in the constructor.